tools/sro2kmm/roles/reboot_nossh/tasks/patch_ds_postreboot.yaml

```diff
@@ -0,0 +1,25 @@
+- name: Patch SRO DaemonSets after reboot
+  k8s:
+    verify_ssl: false
+    api_key: "{{ k8s_auth_results.k8s_auth.api_key }}"
+    api_version: apps/v1
+    kind: DaemonSet
+    state: present
+    namespace: "{{ sro_ds_ns }}"
+    name: "{{ item }}"
+    definition: |
+      spec:
+       updateStrategy:
+         type: OnDelete
+       template:
+         spec:
+          containers:
+            - name: "{{ ds_container_name }}" 
+              lifecycle:
+                preStop:
```

Comment:
Should we also set the postStart hook here?

Reply:
postStart is changed before rebooting here.